std/conv.d

```diff
@@ -2032,6 +2034,54 @@ if (is(T == enum) && !is(S == enum)
     assert(m1 == [[En8143.A, En8143.C], [En8143.C, En8143.A]]);
 }
 
+// issue 20539
+version (StdUnittest)
+{
+    // needs to be outside of the unittest, because else a frame pointer issue
+    // makes it work unintentionally; see next unittest for the version with
+    // the struct being inside
+    struct Test20539
```

Comment:
This struct won‘t have a frame pointer if you declare it as `static`

Reply:
I'm not familiar enough with frame pointers yet. Anyway, this is the original test and it doesn't help to modify it, if that's what you want to do. I just realised that the test passed, when I put the struct inside the unittest. Having had a look, why this is, I found a "could not access frame pointer" problem in the overload using `parse`. So what I did was to add both versions as separate unittests and make sure they both work and do the same.

Reply:
There is no difference between the external or static declaration IIRC (except the visibility of course). This isn't required but this allows to condense bug tests inside one unittest and avoids `version(StdUnittest)`.

Anyway, do as you see fit

Reply:
Ah, sorry, now I got it. I checked that indeed the bug is there, when I replace the global struct with the static one (for extra security, not because I do not believe you). This also helps to remove this stupid `toHash`. Thank you!

Reply:
> [..] (for extra security, not because I do not believe you).[...]

Better safe than sorry

> Thank you!

You're welcome